data/human/culture conversations.txt

```diff
@@ -845,3 +845,84 @@ mission "Republic 300 Year Anniversary"
+				goto ending
+			label known
+			`	The veteran lets out a long-suffering sigh. "Great. At least this one actually knows what they are." The rookie, on the other hand, practically lights up. "They're like space barnacles - until they hatch into wormy bombs! I was just telling him how fun it is to fry 'em!"`
+			`	The veteran gestures around at the cavernous spaceport. "We don't get outbreaks here. This is Delve. They're Delvan Shipworms. They're native. Always have been. They infest the mines, the shipyards, anywhere with exposed metal. That's why every ship gets fumigated before takeoff - to keep 'em from spreading even more than they already have."`
+			`	The rookie eagerly jumps back in. "You ever see one break out of a cocoon in real time? Casing splits and it blasts out like a grenade - it's wild!" The veteran just rubs his face. "Why do I even bother?"`
+			`	With nothing more to add, you make a mental note to give your ship a once-over before takeoff as the workers continue their routine.`
+			label ending
+			`	You give the fumigation teams a final glance and make your way back to your ship, mindful of what might be lurking in the metal unseen.`
+				decline
+
+mission "Exotic Life: Ada"
+	minor
+	source Ada
+	to offer
+		random < 10
+	on offer
+		conversation
+			`You make your way through a dimly lit section of the spaceport, where the usual hum of activity is dampened by an eerie stillness. Large, sealed tarps stretch across the area, some draped over cargo crates, others covering entire sections of the corridor. One enormous tarp seems to be covering a whole ship. Each one bears the same stark warning in bold lettering: 'DELVAN SHIPWORM FUMIGATION.' The air has a faint chemical tang to it, and ahead, two figures move methodically between the tarps.`
+			`	They are covered head to toe in protective suits, thick gloves, and respirators obscuring their faces. One carries a handheld sprayer, releasing a fine mist into the air, while the other checks a monitor strapped to their arm. Their boots crunch softly against the floor as they pass.`
+			choice
+				`	(Ask them what's going on.)`
+				`	(Ignore them and continue on your way.)`
+					decline
+				`	"Nasty little wrigglers, aren't they?"`
+					to display
+						has "exotic animals: delvan shipworm known"
+					goto known
+			`	Curiosity gets the better of you, and you ask what this is all about, gesturing at the tarps. One of them pauses, their voice muffled by the respirator. "Delvan Shipworm outbreak. Nasty infestation. We're exterminating them before they spread to other ships or the city." You can see small scorch marks on the floor nearby - signs of something being burned away.`
+			choice
+				`	"What's a Delvan Shipworm?"`
+				`	"Nasty bombacles, aren't they?"`
+					to display
+						has "exotic animals: delvan shipworm known"
+					goto known
+				`	"Well, good luck with that. I must be on my way."`
```

Comment:
"I must be on my way" sounds awfully formal. Is there a slightly more casual way to say it?

Reply:
As a matter of fact I think there is:
```suggestion
				`	"Well, good luck with that. I should get going."`
```